src/components/Dashboard/ActionRequired.vue

```diff
@@ -7,9 +7,9 @@
           BC Registries is missing information about your business (e.g., business start date, nature of business,
           business address etc.). Please contact BC Registries to input any missing business information. Missing
           information must be entered before you can file changes or dissolve this business.
-          <v-btn class="details-btn" color="primary" text :ripple="false" @click.stop="togglePanel">
-            <v-icon left color="primary">mdi-information-outline</v-icon>
-            <span color="primary">{{ panel === 0 ? "Hide Details" : "View Details" }}</span>
+          <v-btn class="details-btn pl-1" outlined color="primary"  :ripple="false" @click.stop="togglePanel">
+            <v-icon left color="primary" class="mr-0">mdi-information-outline</v-icon>
```

Comment:
Why do you need to remove right margin here? Why do this icon and button have different styling than the exact same buttons in TodoList.vue?

We are already starting to see maintainability problems with duplicated code. This is why I wanted this section built in to TodoList.vue. This is not good. This increases tech debt instead of reducing it.

Reply:
in todo list spacing is wrong i think

![Screen Shot 2022-05-30 at 2 40 59 PM](https://user-images.githubusercontent.com/55506249/171061637-21e77c64-0bea-4f31-8c75-9f9125a87576.png)
I made it like this, to remove spacing between icon and word
![Screen Shot 2022-05-30 at 2 41 33 PM](https://user-images.githubusercontent.com/55506249/171061684-8b4e9bc0-7bda-41ae-aea2-8af88f6e8ae7.png)

I can remove this change if you suggest

Reply:
This is OK for now.

I'm hoping we can simplify this (or make it common with other sub-components) when the Todo List is refactored/cleaned up.